### PR TITLE
bandit happy

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

`# nosec` is added because `urlretrieve` function is used with the hard code url

### Reason for changes

Suppress bandit issue

### Related tickets

ref: 143381

### Tests

build 862

### CODE REVIEW

File: examples/quantization_aware_training/torch/anomalib/main.py [hunk 1]
```diff
@@ -44,11 +44,7 @@ def download_and_extract(root: Path, info: download.DownloadInfo) -> None:
     downloaded_file_path = root / info.url.split("/")[-1]
     print(f"Downloading the {info.name} dataset.")
     with download.DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc=info.name) as progress_bar:
-        urlretrieve(
-            url=f"{info.url}",
-            filename=downloaded_file_path,
-            reporthook=progress_bar.update_to,
-        )
+        urlretrieve(url=f"{info.url}", filename=downloaded_file_path, reporthook=progress_bar.update_to)  # nosec
```
Comment (KodiaqQ) [inline examples/quantization_aware_training/torch/anomalib/main.py:47]: Why can't we use `download_and_extract` as in [OpenVINO example](https://github.com/openvinotoolkit/nncf/blob/develop/examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py?)? Why should we use `urlretrieve`?

**Comment by KodiaqQ** [comment]
Any successful runs?

File: examples/quantization_aware_training/torch/anomalib/main.py (hunk 1 above)
Comment (KodiaqQ) [reply examples/quantization_aware_training/torch/anomalib/main.py:47]: Are there any other options instead of just putting `nosec`?

**Comment by alexsu52** [comment]
> Any successful runs?

build 862

File: examples/quantization_aware_training/torch/anomalib/main.py (hunk 1 above)
Comment (alexsu52) [reply examples/quantization_aware_training/torch/anomalib/main.py:47]: `urlretrieve` is in the black list of bandit https://bandit.readthedocs.io/en/1.7.2/blacklists/blacklist_calls.html, as far I understand, the risk of using `urlretrieve` is in allowing use of ‘file:’’ or custom schemes, but the example uses the hard code url and cann't be specified by user.

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**